symposion/proposals/models.py

```diff
@@ -104,11 +109,12 @@ class ProposalBase(models.Model):
     submitted = models.DateTimeField(
         default=now,
         editable=False,
+        verbose_name=_("Submitted")
     )
-    speaker = models.ForeignKey(Speaker, related_name="proposals")
+    speaker = models.ForeignKey(Speaker, related_name="proposals", verbose_name=_("Speaker"))
     additional_speakers = models.ManyToManyField(Speaker, through="AdditionalSpeaker",
-                                                 blank=True)
-    cancelled = models.BooleanField(default=False)
+                                                 blank=True, verbose_name=_("Addtional speakers"))
+    cancelled = models.BooleanField(default=False, verbose_name=_("Cancelled"))
 
     def can_edit(self):
```

Comment:
It is also necessary to add `__str__` and `@python_2_unicode_compatible` according to #66